specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2018-01-01-preview/schemaregistry.json

```diff
@@ -1,132 +1,224 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "2018-01-01-preview",
-    "title": "schemaregistry",
-    "description": "Description of the new service",
-    "x-ms-code-generation-settings": {
-      "name": "schemaregistryClient"
-    }
-  },
-  "host": "management.azure.com",
-  "schemes": ["https"],
-  "consumes": ["application/json"],
-  "produces": ["application/json"],
-  "security": [
-    {
-      "azure_auth": ["user_impersonation"]
-    }
-  ],
-  "securityDefinitions": {
-    "azure_auth": {
-      "type": "oauth2",
-      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
-      "flow": "implicit",
-      "description": "Azure Active Directory OAuth2 Flow",
-      "scopes": {
-        "user_impersonation": "impersonate your user account"
-      }
-    }
+    "title": "Azure Schema Registry",
+    "version": "2018-01-01-preview"
   },
   "paths": {
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/operations": {
+    "/$schemagroups/getSchemaById/{schema-id}": {
```

Comment:
What is the intention of `$schemagroups` here, is it to be supplied by the caller?  If so you need to specify a parameterized host.  Take a look at [this](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/artifacts.json#L10) for an example.

Reply:
@hmlam

Reply:
$schemagroups is an entry point on the service side to indicate the request is intended for schema group resource. all schema registry request basically will send to that endpoint path. This should not be exposed to customer from SDK prospective though, so should this still be parameterized? what is the benefit or intent of using parameterized host in this static context?

Reply:
I mention parameterized host because the `host` and `schemes` clauses were removed from the baseline swagger.  So at present there is no host URL, just a path.  If the host is static (e.g. `foo.azure.com`) then you need to specify a `host` along with the supported `schemes`.  If callers are expected to provide the host, then use a parameterized host (follow the example in the link I provided earlier).